thundra/thundra_agent.py

```diff
@@ -59,7 +69,7 @@ def __call__(self, original_func):
 
         @wraps(original_func)
         def wrapper(event, context):
-            if self.checkAndHandleWarmupRequest(event):
+            if self.warmup_aware and self.checkAndHandleWarmupRequest(event):
```

Comment:
since your hands are dirty now, can you make this 
`checkAndHandleWarmupRequest` function in snake format?